### ISSUE TITLE
is there any way to use multiple table ref

### ISSUE THREAD

**Issue by nayabshah** [EdisonJpp/react-export-html-table-to-excel#10]
is there any way to use multiple table ref

**Comment by EdisonJpp** [comment]
no, it isn't possible, but you can concatenate the arrays and use `import { downloadExcel } from "react-export-table-to-excel";
`  to export for excel  the data's concatenated

**Comment by cg-flsunshine** [comment]
`   <table id="excel" ref={tableRef}>
        <table></table>
        <table></table>
      </table>` 
      try like this
